### DISCUSSION TITLE
Getting issue when trying to build in Android

### DISCUSSION THREAD

**Discussion by Softkeydel** [TheWidlarzGroup/react-native-video#4169] [Q&A] [answered]
### What are you trying to do?

Build the app on Android with react-native-video, but the build fails with a `ReactVideoPackage not found` error. Please suggest a fix.

### What have you tried?

Build configuration:

- minSdk 24
- compileSdk 34
- targetSdk 34
- Gradle distribution: `https://services.gradle.org/distributions/gradle-7.5.1-all.zip`
- react 18.2.0
- react-native 0.72.6
- react-native-video 6.5.0

![hello_042945](https://github.com/user-attachments/assets/fe58b6d0-6e9b-41c6-a5c6-94bc35c9b9c2)

### Relevant code

```
com.brentvatne.react.test.ReactVideoPackage not found
```

### Version

v6

### Platforms

Android

**Comment by freeboub** [comment, marked as answer]
React.test shouldn't be used ... Please review and clean your gradle integration (especially if you come from 5.2)

**Reply by Softkeydel** [reply]
@freeboub 

https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
This gradle version is being used.
Each times I've cleaned gradle build and try to make new build, I'm getting the same issue
